src/commands/copy/copy-worker.ts

```diff
@@ -13,21 +15,31 @@ import { registerCli } from '../common.ts';
 import { isTiff } from '../tileindex-validate/tileindex.validate.ts';
 import type { CopyContract, CopyContractArgs, CopyStats } from './copy-rpc.ts';
 
-const Q = new ConcurrentQueue(10);
+const numberOfConcurrentItems = Math.max(cpus().length - 2, 1);
+const Q = new ConcurrentQueue(numberOfConcurrentItems); // saturate all CPUs and leave 2 cores free for the system
+export const MIN_SIZE_FOR_COMPRESSION = 500; // testing with random ASCII data shows that compression is not worth it below this size
```

Comment:
in this repo consts are `TitleCase` I wonder why the linter is not picking this up.

Reply:
Why is `TitleCase` used for exported consts in this repo rather than `SCREAMING_SNAKE_CASE`?

Reply:
There is no real reason other than that is what is done in this repo, you can check other files searching for `export const` to see what things are exported as.

Reply:
I just wondered why, but it's not important :)